src/server/standalone.js

```diff
@@ -431,7 +431,8 @@ function StandAloneServer(gmeConfig) {
                                 req.userData = {
                                     token: newToken,
                                     newToken: true,
-                                    userId: result.content.userId
+                                    userId: result.content.userId,
+                                    displayName: result.content.displayName
```

Comment:
Right now the inferred users with displayName are confined to tokens in the query..